plugin/ochttp/propagation/tracecontext/propagation.go

```diff
@@ -98,13 +104,26 @@ func (f *HTTPFormat) SpanContextFromRequest(req *http.Request) (sc trace.SpanCon
 	return sc, true
 }
 
+func getRequestHeader(req *http.Request, name string, commaSeparated bool) (hdr string, ok bool) {
+	ok = false
```

Comment:
It's not really clear what "ok" means here. I would expect it to mean that the header is present, but that doesn't seem to be the case since it can be false even when the header is present. Perhaps it should have a different name, or be documented.